packages/core/src/App/Containers/WelcomeModal/welcome-modal.jsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import { withRouter } from 'react-router';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Modal, Icon, Button, ThemedScrollbars } from '@deriv/components';
+import { routes } from '@deriv/shared';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+
+const WelcomeColumn = ({
+    button_text,
+    className,
+    description,
+    icons,
+    is_hovered,
+    onButtonClick,
+    onMouseEnter,
+    onMouseLeave,
+    platforms,
+    title,
+}) => {
+    return (
+        <div
+            className={classNames('welcome-column', className)}
+            onMouseEnter={onMouseEnter}
+            onMouseLeave={onMouseLeave}
+        >
+            <div className='welcome-column__icons'>
+                {icons.map((icon, index) => (
+                    <Icon className='welcome-column__icon' icon={icon} key={index} size={48} />
+                ))}
+            </div>
+            <div className='welcome-column__title'>{title}</div>
+            <div className='welcome-column__description'>{description}</div>
+            <div className='welcome-column__platforms'>
+                <p className='welcome-column__platforms__title'>{localize('Platforms')}</p>
+                <div className='welcome-column__platforms__container'>
+                    {platforms.map((platform, index) => (
+                        <div className='welcome-column__platform' key={index}>
+                            <Icon className='welcome-column__platform__icon' icon={platform.icon} size={32} />
+                            <h3 className='welcome-column__platform__title'>{platform.title}</h3>
+                            <p className='welcome-column__platform__description'>{platform.description}</p>
+                        </div>
+                    ))}
+                </div>
+            </div>
+            <Button
+                className='welcome-column__button'
+                onClick={onButtonClick}
+                large
+                primary={is_hovered}
+                secondary={!is_hovered}
+            >
+                {button_text}
+            </Button>
+        </div>
+    );
+};
+
+WelcomeColumn.propTypes = {
+    button_text: PropTypes.string,
+    description: PropTypes.string,
+    icons: PropTypes.array,
+    onButtonClick: PropTypes.func,
+    platforms: PropTypes.arrayOf(PropTypes.object),
+    title: PropTypes.string,
+};
+
+const WelcomeModal = ({ toggleWelcomeModal, history }) => {
+    const [hovered, setHovered] = React.useState(null);
+
+    const switchToRealAccount = React.useCallback(() => {
+        toggleWelcomeModal(false);
+        history.push(routes.trade);
+    }, [toggleWelcomeModal, history]);
+
+    const switchToDMT5 = React.useCallback(() => {
```

Comment:
this is very nice!, btw, can combine this function and get route as argument

```suggestion
    const switchAccount = React.useCallback(({ route }) => {
      toggleWelcomeModal(false);
        history.push(route);
    }
```